test/quantization/test_quantize_script.py

```diff
@@ -1170,6 +1170,38 @@ def test_qconfig_with_types(self):
                                  'sub2.fc2': default_qconfig, 'fc3': default_qconfig}
         self.assertEqual(qconfig_dict, expected_qconfig_dict)
 
+    def test_module_list(self):
+        class SimpleLinearLayer(torch.nn.Module):
+            def __init__(self):
+                super(SimpleLinearLayer, self).__init__()
+                self.fc = torch.nn.Linear(5, 5).float()
+
+            def forward(self, x):
+                return self.fc
+
+        class ComplexModel(torch.nn.Module):
+            def __init__(self):
+                super(ComplexModel, self).__init__()
+                self.layers = torch.nn.ModuleList([SimpleLinearLayer() for i in range(2)])
+
+            def forward(self, x):
+                # type: (torch.Tensor) -> List[torch.Tensor]
+                states = []
+                for layer in self.layers:
+                    val = layer.fc(x)
+                    states.append(val)
+                return states
+
+        data = torch.rand((1, 5), dtype=torch.float)
+        qconfig_dict = {'': default_qconfig}
+        model = torch.jit.script(ComplexModel()).eval()
+        model = prepare_script(model, qconfig_dict)
+        assert len(attrs_with_prefix(model, '_observer')) == 3
+        model(data)
+        model = convert_script(model, debug=False)
+        FileCheck().check_count("quantized::linear", 2, exactly=True) \
+                   .run(model.graph)
```

Comment:
I don't understand how this test is failing with this message. The output graph seems file. Why is it looking for CHECK_NOT when I specified check_count. And why only on macos? @jerryzh168 any ideas?
```
Apr 23 14:51:35 ERROR [0.145s]: test_module_list (quantization.test_quantize_script.TestQuantizeScriptJitPasses)
Apr 23 14:51:35 ----------------------------------------------------------------------
Apr 23 14:51:35 Traceback (most recent call last):
Apr 23 14:51:35   File "/Users/distiller/project/test/quantization/test_quantize_script.py", line 1203, in test_module_list
Apr 23 14:51:35     .run(model.graph)
Apr 23 14:51:35 RuntimeError: Expected to not find "quantized::linear" but found it
Apr 23 14:51:35   %val.2.dequant.0 : Tensor = aten::dequantize(%21)
Apr 23 14:51:35   %23 : Tensor[] = aten::append(%states.1, %val.2.dequant.0) # /Users/distiller/project/test/quantization/test_quantize_script.py:1192:20
Apr 23 14:51:35   %31 : Tensor = quantized::linear_prepack(%63, %self.layers.1.fc.bias)
Apr 23 14:51:35                  ~~~~~~~~~~~~~~~~~ <--- HERE
Apr 23 14:51:35   %35 : Tensor = quantized::linear(%x.1.quant, %31, %self.val.4_scale_0, %self.val.4_zero_point_0)
Apr 23 14:51:35   %val.4.dequant.0 : Tensor = aten::dequantize(%35)
Apr 23 14:51:35 From CHECK-NOT: quantized::linear
```

https://dr.pytorch.org/api/view-log-full?build_id=113707339